madlibs.js

```diff
@@ -1,42 +1,108 @@
-/**
- * Complete the implementation of parseStory.
- * 
- * parseStory retrieves the story as a single string from story.txt
- * (I have written this part for you).
- * 
- * In your code, you are required (please read this carefully):
- * - to return a list of objects
- * - each object should definitely have a field, `word`
- * - each object should maybe have a field, `pos` (part of speech)
- * 
- * So for example, the return value of this for the example story.txt
- * will be an object that looks like so (note the comma! periods should
- * be handled in the same way).
- * 
- * Input: "Louis[n] went[v] to the store[n], and it was fun[a]."
- * Output: [
- *  { word: "Louis", pos: "noun" },
- *  { word: "went", pos: "verb", },
- *  { word: "to", },
- *  { word: "the", },
- *  { word: "store", pos: "noun" }
- *  { word: "," }
- *  ....
- * 
- * There are multiple ways to do this, but you may want to use regular expressions.
- * Please go through this lesson: https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/regular-expressions/
- */
+let objectsArray = [];
+//This function will be used for alert 
+function onlyLetters(str) {
+  return /^[A-Za-z]*$/.test(str);
+}
 function parseStory(rawStory) {
-  // Your code here.
-  return {}; // This line is currently wrong :)
+  let storyArr = rawStory.split(" ");
```

Comment:
```suggestion
  const storyArr = rawStory.split(" ");
```
Always use `const` instead of `let` unless you plan to change the variable later down the line.